### ISSUE TITLE
Brainstorming paper ideas

### ISSUE THREAD

**Issue by lwjohnst86** [UofTCoders/paper-rcourse#1]
See Issue [#13](https://github.com/UofTCoders/rcourse/issues/13) for the start of these discussions.

**Comment by lwjohnst86** [comment]
Some ideas on the main focus of the paper:

1. As a tool/process for other groups to try and emulate
2. As an experience of a student group to get a course incorporated into a university curriculum
3. As an argument to show that hands on, project focused courses should be the primary types of style that upper level courses have, based on the student feedback
4. As resources for how to create a hands on data analysis course, a sorta "lessoned" learned.

We could maybe vote (or narrow down or add to) on which focus we should be going with.

I personally like 1 and 4. More on the side of "hey, this is what we did, here's how you can do it and learn from our mistakes" type thing.

**Comment by joelostblom** [comment]
Sorry for the late reply to this.

I like the "hey, this is what we did, here's how you can do it and learn from our mistakes" approach, so I guess mostly 1 and 4. I think aspects from bullet 3 will be part of the introduction.

I don't know if we will talk about bullet 2 much/at all, depends on how specific we are regarding the process of getting the course pilot approved at EEB, as in how much of the internal process would we and EEB be comfortable having published in a paper (there might be some restrictions of that?). But I guess the next step here would be for us to ouline/list specific topics and we can discuss that then.

@linamnt @mbonsma @QuLogic @lcoome What are your thoughts on the main ideas that should go into this paper?

**Comment by mbonsma** [comment]
I like 1, 3, and 4 the best, and probably 1 and 4 best. 

I agree that 2 is probably too specific to the institution to be relevant more broadly. We can mention it in the sense that this is a curriculum change initiated by students, but I wouldn't want to get into it too much.

**Comment by linamnt** [comment]
I think 1/4 are most practical. I like 3 but as Joel mentioned, we can use it as a discussion/intro point in either 1 or 4. 
Out of 1 and 4, I prefer 4 mainly because it could be more broadly applied.

**Comment by mbonsma** [comment]
I think of 1 and 4 as being essentially the same thing, but maybe I'm missing the difference.

**Comment by linamnt** [comment]
Ah I guess I thought of 1 as more of a 'use as is' and 4 as more of how to tailor to the desired topics, Not sure if Luke had other thoughts for those two. But they do sound pretty much the same and there's nothing really stopping us from doing both?

**Comment by lwjohnst86** [comment]
Yes, 1 and 4 are pretty similar. Again, these were *brainstorming* ideas, so there would be some overlap. 

However, I see the focus of the paper being slightly different between the two. 1 is more of a "formalized" process/workflow to doing quantitative/hands on course, while 4 is more of a "this is our experience and what we learned". Does that make sense? Of course, if we go either route, there will be overlap, since they are very similar. But again, we can refine the specific aim of the paper as we go.

**Comment by joelostblom** [comment]
@lwjohnst86 Yes, I think that makes sense, and I believe we can include both. This is how I think of it:

- Intro can include 3, why we use this type of hands on project style.
- Results will include 4, our lessons learned (and reiterate that point 3 was popular in the student feedback).
- Conclusions/Recommendations will include 1, based on our experience, here is a formal process for creating such a course.

**Comment by SaraMati** [comment]
the paper that mentions UofT coders  
[Building a local community of practice in scientific programming for Life Scientists](https://www.biorxiv.org/content/early/2018/11/11/265421)

**Comment by lwjohnst86** [comment]
Similar paper: https://www.biorxiv.org/content/10.1101/666768v1.abstract?%3Fcollection=
